package model.SaveAndLoadTest;

import model.log.Log;
import org.junit.Assert;
import org.junit.Test;

import java.io.IOException;
import java.util.ArrayList;
import java.util.Date;
import java.util.Map;

public class LogTest {

    @Test
    public void saveLog() throws IOException {
        //TODO: TEST IT, IT HAS A PROBLEM
//        ArrayList<String> productsName = new ArrayList<>();
//        productsName.add("aaaaa");
//        Date date = new Date();
//        Log log = new Log(date, 100, 20, productsName, "Amirali",
//                true);
//        String idBeforeSaving = log.getId();
//        Log.saveData();
//        Log.getLogs().remove(idBeforeSaving);
//        Log.loadData();
//        Assert.assertEquals(log.toString(), Log.getLogById(idBeforeSaving).toString());
    }
}
